CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+Workarea Sitemaps 2.1.8 (2020-07-01)
```

Comment:
Please remove this change, this get generated dynamically when we run the release task